### DISCUSSION TITLE
Convert multiple svg or png to ico

### DISCUSSION THREAD

**Discussion by chdeto** [ImageMagick/ImageMagick#5477] [Help]
Hello,

I have a long time problem converting preferably multiple .svg to .ico format (.png to .ico work as well). 

I am using Mac. I am originally a designer so please treat this a a beginner comment. I would like to convert the software icons to .ico so developer colleagues can just place them automatically. The issue with different tools I tried is that the end quality of the icon is very poor. 

Do you have any suggestions. This is what I am trying but the .ico file ends up being empty. 

I installed HomeBew and ImageMagic 

I am running this command: convert "file names (individual png's)" -colors 256 output.ico 
The file comes up as empty; I know I used this in the past but probably I am making a rookie mistake. However as I remember this also didn't produce high quality icon.

Thank you for your help.

**Comment by canonex** [comment]
You can try something like:
`convert -density 1200 -background none in.svg out.ico`

- the **density** indicates how many pixels per inch, the default is 72. Use a value good for you.
- the **-background none** indicates if you need a transparent background. Remove the option to have a white background.
